luarules/gadgets/unit_paralyze_damage_limit.lua

```diff
@@ -48,6 +48,11 @@ local spGetUnitHealth = Spring.GetUnitHealth
 
 function gadget:UnitPreDamaged(uID, uDefID, uTeam, damage, paralyzer, weaponID, projID, aID, aDefID, aTeam)
     if paralyzer then
+	
+
+
+	
+	
```

Comment:
Why add so many blank lines?

Reply:
I had a hook there originally, it was moved later, so the lines are just debris that surrounded it